v2/tools/generate-techniques-documentation.go

```diff
@@ -3,14 +3,16 @@ package main
 import (
 	"bytes"
 	"fmt"
-	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
-	_ "github.com/datadog/stratus-red-team/v2/pkg/stratus/loader"
-	"github.com/datadog/stratus-red-team/v2/pkg/stratus/mitreattack"
 	"log"
 	"os"
 	"path/filepath"
 	"strings"
 	"text/template"
+
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
+	_ "github.com/datadog/stratus-red-team/v2/pkg/stratus/loader"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus/mitreattack"
+	"gopkg.in/yaml.v3"
```

Comment:
VSCode uses `go fmt` and it re-ordered the import like that (I belive that's `goimports` default configuration`. Let me know if you would like the import the way it was and I will edit it using vim.